app/views/grantees/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="details-header">
+  <div class="grid-container">
+    <h1><%= @grantee.name %></h1>
+    <h2>Grantee #<%= @grantee.id %>, Region <%= @grantee.region %></h2>
+  </div>
+</div>
+<div class="details-body grid-container">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-4">
+      <div class="summary-section">
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+        </div>
+      </div>
+    </div>
+    <div class="grid-col-8">
+      <ul class="usa-process-list">
```

Comment:
[suggestion] it looks like this is just a list of complaints, rather than the complaint containers. (if this feels like it's gotten too big, though, this can be done in the next card, though)

Reply:
@lauranash18f is going to give me some guidance on the component layout